server.py

```diff
@@ -54,6 +55,7 @@ def dashboard():
 
 @app.route('/api/<endpoint>', methods=['GET'])
 @require_authorization
+@ratelimits.ratelimit

```

Comment:
How about `@ratelimit` instead? Mostly for simplicity and it looks neater.

Reply:
Could do, would simply have to change up import statement

Reply:
Done